README.md

```diff
@@ -1,6 +1,159 @@
-# IntelliTect
+# IntelliTect Open-Source Libraries
 
 [![Build status](https://ci.appveyor.com/api/projects/status/d69509l4p7gyb77j?svg=true)](https://ci.appveyor.com/project/IntelliTect/intellitect)
 
 
-Libraries of useful things, focused on shipping Nuget packages.
+Libraries of useful C# things, focused on shipping Nuget packages.
+
+Libraries
+=========
+
+* IntelliTect.Utilities [![NuGet](https://img.shields.io/nuget/v/IntelliTect.Utilities.svg)](https://www.nuget.org/packages/IntelliTect.Utilities/)
+    - AssemblyInfo: Gets an assembly's linker date/time.
+
+Contributing
+============
+
+* [Prerequisites](#prerequisites)
+* [Code](#code)
+  * [Code style](#code-style)
+  * [Dependencies](#dependencies)
+  * [Unit tests](#unit-tests)
+* [Contributing process](#contributing-process)
+  * [Get buyoff or find open community issues or features](#get-buyoff-or-find-open-community-issues-or-features)
+  * [Set up your environment](#Set-up-your-environment)
+  * [Prepare commits](#prepare-commits)
+  * [Submit pull request](#Submit-pull-request)
+  * [Respond to feedback on pull request](#respond-to-feedback-on-pull-request)
+* [Other general information](#other-general-information)
+* [Acknowledgement](#acknowledgement)
+
+## Prerequisites
+
+By contributing to IntelliTect, you assert that:
+
+* The contribution is your own original work.
+* You have the right to assign the copyright for the work (it is not owned by your employer, or
+  you have been given copyright assignment in writing).
+* You [license](https://github.com/IntelliTect/IntelliTect/blob/master/LICENSE) the contribution under the terms applied to the rest of the IntelliTect project.
+
+## Code
+### Code style
+
+Normal .NET coding guidelines apply.
+See the [Framework Design Guidelines](https://msdn.microsoft.com/en-us/library/ms229042%28v=vs.110%29.aspx) for more information.
+
+### Dependencies
```

Comment:
We might need to change this section in the future since I expect that we will have multiple project with varying dependencies.